index.css

```diff
@@ -0,0 +1,75 @@
+body{
```

Comment:
I think more could've been done if you had more time for this assignment. Here are few feedbacks: 
1. Nice touch with favicon. Definitely important to include them in future projects as well. 
2. Good use of flex for main div and also centering background. 
3. You could use inline or similar CSS methods for the footer. 
4. Layouts are generally in right place but they are lacking CSS. 
As for next assignments after Unit 2 assessment, I want to see more efforts and time to style your page. Let me know if you have any questions or assistance.